src/utils.js

```diff
@@ -8,3 +8,31 @@ exports.findHighest = (...numbers) => {
     return highest;    
 };
 
+
+exports.findAverage = (...numbers) => {
```

Comment:
The readability of the code can be improved. One way to achieve this is to use the concept of an early return. In that case we can update our code to this:

```
const findAverage = (...numbers) => {
  if (numbers.length < 1) {
    throw new Error("This function expects at least one argument")
  }

  if (!numbers.every((element) => Number.isInteger(element))) {
    throw new Error("The function expects an integer")
  }

  const average = numbers.reduce((a,b) => a + b, 0) / numbers.length;

  if (Number.isInteger(average)) {
    return average;
  } else {
    return parseFloat(average.toFixed(2));
  }
}
```

Reply:
ok